source/JsMaterialX/test/codeExamples.spec.js

```diff
@@ -84,9 +84,9 @@ describe('Code Examples', () => {
 
         // Read a document from disk.
         const doc = mx.createDocument();
-        const readOptions = new mx.XmlReadOptions();
-        readOptions.applyFutureUpdates = true;
-        mx.readFromXmlString(doc, xmlStr, readOptions);
+        // const readOptions = new mx.XmlReadOptions();
```

Comment:
why is this commented out? is it not necessary or should it be uncommented later?